example/20-componentconfig-gardenlet.yaml

```diff
@@ -118,7 +118,6 @@ featureGates:
   HVPAForShootedSeed: true
   ManagedIstio: true
   APIServerSNI: true
-  ReversedVPN: true
```

Comment:
https://github.com/gardener/gardener/blob/f723801e7e2fe7e07b247402c28da60ccc0d55a8/extensions/pkg/webhook/controlplane/genericmutator/mutator.go#L65-L67 has to be adapted.

Reply:
Adapted.